controllers/freebasics/forms.py

```diff
@@ -0,0 +1,53 @@
+from controllers.docker.forms import DockerControllerForm
+from controllers.freebasics.models import FreeBasicsController
+from django import forms
+from django.forms.utils import ErrorList
+
+TEMPLATE_CHOICES = (
```

Comment:
Actually, can you move this to `models.py` as part of the `FreeBasicsController` and use that as the choices there too. You can reference it from here using `FreeBasicsController.TEMPLATE_CHOICES`